bin/anthology/papers.py

```diff
@@ -223,7 +231,10 @@ def get_booktitle(self, form="xml"):
 
         See `get_title()` for details.
         """
-        return self.formatter(self.get("xml_booktitle"), form)
+        if 'xml_booktitle' in self.attrib:
+            return self.formatter(self.get("xml_booktitle"), form)
+        elif self.parent_volume is not None:
+            return self.parent_volume.get('title')
```

Comment:
This function can potentially return nothing... it should either return an empty string if both if-statements fail, or throw an Exception.

Reply:
Done.